test/persistence-hooks.suite.js

```diff
@@ -86,6 +86,18 @@ module.exports = function(dataSource, should, connectorCapabilities) {
           });
       });
 
+      it('should trigger access hook once, if near filter is used', function(done) {
```

Comment:
How about the following name?

```js
it('triggers correct hooks when near filter is used', ...)
```

This way the test can be easily extended later when we fix the implementation to trigger the "loaded" hook too.